src/modules/processing/url.js

```diff
@@ -23,15 +23,15 @@ export function aliasURL(url) {
                 ** but we only care about the 1st segment of the path */
                 url = new URL(`https://youtube.com/watch?v=${
                     encodeURIComponent(parts[1])
-                }`)
+                    }`)
             }
             break;
 
         case "pin":
             if (url.hostname === 'pin.it' && parts.length === 2) {
                 url = new URL(`https://pinterest.com/url_shortener/${
                     encodeURIComponent(parts[1])
-                }`)
+                    }`)
```

Comment:
why?

Reply:
it seem I accidentally do auto-formating with my IDE